.github/workflows/ci.yml

```diff
@@ -21,11 +21,11 @@ env:
 jobs:
     test_pgsql:
         name: "Test / PostgreSQL"
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-20.04
         strategy:
             fail-fast: false
             matrix:
-                php: [ '7.3', ]
+                php: [ '7.4', ]
```

Comment:
In the followup PR that introduces standalone testing please also make sure we test 7.4+8.0